h2o-automl/src/main/java/ai/h2o/automl/Leaderboard.java

```diff
@@ -111,17 +116,21 @@ private Leaderboard() {
   /**
    *
    */
-  private Leaderboard(String project_name, UserFeedback userFeedback, Frame leaderboardFrame) {
+  public Leaderboard(String project_name, UserFeedback userFeedback, Frame leaderboardFrame) {
```

Comment:
Why public?

Reply:
Because it's now called from the LeaderBoardHandler in the case that the Leadboard is null.  This can happen do to a race: the AutoML object is created and immediately the client queries before the AutoML object's Leaderboard has been created.